packages/terra-table/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-table",
   "main": "lib/index.js",
-  "version": "4.36.0",
+  "version": "4.37.0",
```

Comment:
This needs to be a major alpha release. So 5.0.0-alpha (or however the correct format is).

Reply:
Should be correct now [73f2111](https://github.com/cerner/terra-framework/pull/1838/commits/73f21115e3c5e98c7d9ea6f365fffa77ed4a87b4)

Reply:
Looks good!